ui/pages/settings/networks-tab/networks-tab.constants.js

```diff
@@ -51,6 +53,15 @@ const defaultNetworksData = [
     ticker: TEST_NETWORK_TICKER_MAP[GOERLI],
     blockExplorerUrl: 'https://goerli.etherscan.io',
   },
+  {
+    labelKey: SEPOLIA,
+    iconColor: '#CFB5F0',
```

Comment:
@georgewrmarshall Do you know if we can use `var` colors for these variables?

Reply:
Nice catch @darkwing! It's in the works https://github.com/MetaMask/design-tokens/issues/238. I can update this once we have updated the design tokens repo :)